src/client/pages/OfferDebugger/components/QuoteData.tsx

```diff
@@ -160,13 +162,30 @@ const getSwedishAccidentQuoteValues = (values: any) => {
     data: {
       ...quoteTypeValues,
       type: QuoteType.SwedishAccident,
-      student:
+      isStudent:
         subType === ApartmentType.StudentBrf ||
         subType === ApartmentType.StudentRent,
     },
   }
 }
 
+const getSwedishHouseQuoteValues = (values: any) => {
+  const { data: swedishApartmentData, ...holderData } = values
+  const { subType: _, ...quoteTypeValues } = swedishApartmentData
+  return {
+    ...holderData,
+    data: {
+      ...quoteTypeValues,
+      type: QuoteType.SwedishHouse,
+      ancillaryArea: 0,
+      yearOfConstruction: 1977,
+      numberOfBathrooms: 2,
+      extraBuildings: [],
+      isSubleted: false,
```

Comment:
We should probably add all these things to the `QuoteCart` query - currently we only ask for `street` back.